### PR TITLE
[Doc] Add Chinese documentation of coord_sys_tutorial and fix some links

### PR DESCRIPTION
**Pull request by Xiangxu-0103**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation
Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification
Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)
Does the modification introduce changes that break the back-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)
If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: docs/zh_cn/tutorials/coord_sys_tutorial.md [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+  |    |____|____|
+  |         dx
+  |
+```
+
+## 与支持的数据集的原始坐标系的关系
+
+### KITTI
+
+KITTI 数据集的原始标注是在相机坐标系下的，详见 [get_label_anno](https://github.com/open-mmlab/mmdetection3d/blob/v1.0.0.dev0/tools/data_converter/kitti_data_utils.py)。在 MMDetection3D 中，为了在 KITTI 数据集上训练基于激光雷达的模型，首先将数据从相机坐标系转换到激光雷达坐标系，详见 [get_ann_info](https://github.com/open-mmlab/mmdetection3d/blob/v1.0.0.dev0/mmdet3d/datasets/kitti_dataset.py)。为了训练基于视觉的模型，数据保持在相机坐标系下不变。
```
Comment (Tai-Wang) [inline docs/zh_cn/tutorials/coord_sys_tutorial.md:135]: 可能把这些 link 都改成 master 里对应的？

File: docs/zh_cn/tutorials/coord_sys_tutorial.md [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+
+我们将各种框的旋转设定为绕着重力轴逆时针旋转。因此，为了旋转一个 3D 框，我们首先计算新框的中心，然后将旋转角度添加到转向角。
+
+详见[此处](https://github.com/open-mmlab/mmdetection3d/blob/v1.0.0.dev0/mmdet3d/core/bbox/structures/cam_box3d.py)代码了解更多细节。
+
+## 常见问题
+
+#### Q1: 与框相关的算子是否适用于所有坐标系类型？
+
+否。例如，在该[文件夹](https://github.com/open-mmlab/mmdetection3d/blob/v1.0.0.dev0/mmdet3d/ops/roiaware_pool3d)下的算子只适用于深度或激光雷达坐标系。由于如果从上方看，旋转是顺时针的，所以 KITTI 数据集[这里](https://github.com/open-mmlab/mmdetection3d/blob/v1.0.0.dev0/mmdet3d/core/evaluation/kitti_utils)的评估函数仅适用于相机坐标系。
```
Comment (Tai-Wang) [inline docs/zh_cn/tutorials/coord_sys_tutorial.md:218]: 可以具体下文件夹是什么的算子，比如：\[用于 RoI-Aware Pooling 的算子](xxxxxx)

Comment (Tai-Wang) [reply docs/zh_cn/tutorials/coord_sys_tutorial.md:218]: 因为目前 op 移到 mmcv，可以改成 mmcv 里面相应的链接？

**Comment by ZwwWayne** [comment]
Hi @Xiangxu-0103 ,
Could you update your documentation from this https://github.com/open-mmlab/mmdetection3d/pull/1001/files#diff-665454c8e80f2d9e2f1fb4cdfb0830faf3d9287acd3aae78064a61d58db51078? We find that previously we have a version of the doc, but it is lost somehow.

**Comment by Xiangxu-0103** [comment]
> Hi @Xiangxu-0103 , Could you update your documentation from this https://github.com/open-mmlab/mmdetection3d/pull/1001/files#diff-665454c8e80f2d9e2f1fb4cdfb0830faf3d9287acd3aae78064a61d58db51078? We find that previously we have a version of the doc, but it is lost somehow.

The documentation has been updated according to PR #1001.

File: docs/zh_cn/tutorials/coord_sys_tutorial.md [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+__|____|____|____|______\ x 右
+  |    |    |    |      /
+  |    |    |    |
+  |    |____|____|
+  |
+```
+
+## 框尺寸的定义
+
+框尺寸的定义与转向角 (yaw) 的定义是分不开的。在上一节中，我们提到如果一个框的转向角 (yaw) 为 0，它的方向就被定义为与 x 轴平行。那么自然地，一个框对应于 x 轴的尺寸应该是 ``$$`dx`$$``。但是，这在某些数据集中并非总是如此（我们稍后会解决这个问题）。
```
Comment (jshilong) [inline docs/zh_cn/tutorials/coord_sys_tutorial.md:95]: 解决? or 解释?

**Review by jshilong — APPROVED** [review]
LGTM

File: docs/zh_cn/tutorials/coord_sys_tutorial.md (hunk 3 above)
Comment (Xiangxu-0103) [reply docs/zh_cn/tutorials/coord_sys_tutorial.md:95]: According to the english tutorial, this is the problem that needs to be addressed in the future, so I think it should be translated as `解决`.

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**